docs/useRemoveFromStore.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: "useRemoveFromStore"
+---
+
+# `useRemoveFromStore`
+
+This hook allows to remove a value from the [Store](./Store.md). 
+
+## Syntax
+
+```jsx
+import { useRemoveFromStore } from 'react-admin';
+
+const remove = useRemoveFromStore();
+remove(key);
+```
+
+The `key` should be a string, and is used to access local storage.  It can be passed either when calling the hook, or when calling the callback:
```

Comment:
```suggestion
The `key` should be a string, and is used to access local storage. It can be passed either when calling the hook, or when calling the callback:
```